### DISCUSSION TITLE
Update frequency

### DISCUSSION THREAD

**Discussion by doniks** [hodcroftlab/covariants#178] [General]
I see the data points are once per week, But now the latest one is May 31, which is almost three weeks now. 

What is the update frequency? Is there a problem recently?

**Comment by emmahodcroft** [comment]
Hi @doniks - The data points are always behind because for most countries it takes 2 weeks minimum from sample to sequence being online - but it can often be longer. So no updates ever go up to the current date! 

I try to update 2-3 times a week when I can, but sometimes I manage only once a week if I'm very busy or if there are technical issues. Unfortunately this week we had both technical issues and a code update (about to go live) so I only updated once. I'm hoping to get back to more regular schedule this week.

I realise it's a bit frustrating when updates don't come as regularly, but unfortunately the website is scientifically a one-woman project, so I appreciate everyone's patience when I can't get it done as often as I'd like!

**Reply by tlearyus** [reply]
Was looking for the data for the two new variants of concern B.1.621 and C.1.2. 

Any idea when this will be added?

**Reply by emmahodcroft** [reply]
Hi @tlearyus ! Actually, neither of these are VoC as [recognised by the WHO](https://www.who.int/en/activities/tracking-SARS-CoV-2-variants/). However, B.1.621 was announced recently as a Variant of Interest (VoI) - "Mu". As you can see from the chart on the link above, this is "21H" in Nextstrain nomenclature - so you can find it on the CoVariants page as "[21H](https://covariants.org/variants/21H)"!

I'll be updating CoVariants to reflect the change in status ASAP.

C.1.2 is not currently recognised as VoI by the WHO, and I'm still deciding whether to add it to CoVariants!
